.gitignore

```diff
@@ -2,3 +2,4 @@ RAMCloud
 RAMCloud-install
 __pycache__/
 *.py[cod]
+tmp/
```

Comment:
This keeps showing up.

Reply:
This time it's intentional. In cluster_test_utils.py, the method outputLogs() has path variable set to "/src/tmp"by default. I.e., you'd get a tmp directory if you use default behavior. I couldn't find a better name for the default output logs directory.